core/src/main/java/org/commonjava/maven/ext/core/ManipulationManager.java

```diff
@@ -191,6 +192,10 @@ public void init( final ManipulationSession session )
     public void scanAndApply( final ManipulationSession session )
                     throws ManipulationException
     {
+        final PreparseGroovyManipulator preparseGroovyManipulator = new PreparseGroovyManipulator( null, null, pomIO );
```

Comment:
Manual creation means the injection won't work. I think this needs changing. This also affects my comments further down regarding your manual creation of fileIO etc.